client/src/pages/CreateNewPage.js

```diff
@@ -0,0 +1,22 @@
+import "./CreateNewPage.css";
+
+export default function CreateNewPage() {
+  return (
+    <div className="Form">
+      <div className="Page__heading">
+        <h2>Add a recipe</h2>
+      </div>
+
+      <form className="form">
+        <input type="text" id="name" name="name" placeholder="cake name" />
+        <input
```

Comment:
when you want a multiple line input, the `<textarea>` element would be a better choice.